Templating.sln

```diff
@@ -28,6 +28,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{342602FC-9
 		test\TemplateTests.props.in = test\TemplateTests.props.in
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DotNetToolsInstaller", "DotNetToolsInstaller\DotNetToolsInstaller.csproj", "{89317EDE-A5B9-4688-A74A-DC9FBDA71438}"
```

Comment:
Nit: maybe put this under the tests folder?

Reply:
Ah good point. Doesn't deserve its own top-level folder.